sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1130,7 +1132,25 @@ def __init__(
         'fnapi-step-%s' % self.process_bundle_descriptor.id,
         self.counter_factory)
 
-    self.ops = self.create_execution_tree(self.process_bundle_descriptor)
+    with concurrent.futures.ThreadPoolExecutor(
+        max_workers=1, thread_name_prefix='ExecutionTreeCreator') as executor:
+      future = executor.submit(
+          self.create_execution_tree, self.process_bundle_descriptor)
+      try:
+        self.ops = future.result(timeout=3600)
+      except concurrent.futures.TimeoutError:
+        # In rare cases, unpickling a DoFn might get permanently stuck,
+        # for example when unpickling involves importing a module and
+        # a subprocess is launched during the import operation.
+        _LOGGER.error(
+            'Timed out when creating execution tree for %s.\n%s',
+            self.process_bundle_descriptor.id,
```

Comment:
since we terminate this sdk harness later, I think this error should be harmless. So can we mention this somehow?